drill-admin/src/main/kotlin/com/epam/drill/endpoints/AgentManager.kt

```diff
@@ -63,10 +63,10 @@ class AgentManager(override val kodein: Kodein) : KodeinAware {
             agentInfoDb.toAgentInfo()
         } else {
             AgentInfoDb.new(agentId) {
-                name = "-"
+                name = agentId
                 status = AgentStatus.NOT_REGISTERED
-                groupName = "-"
-                description = "-"
+                groupName = "n/a"
```

Comment:
Is there any specific purpose to persist value "n/a"?

Reply:
Drill specification